ArduPlane/quadplane.cpp

```diff
@@ -2228,11 +2238,14 @@ void QuadPlane::vtol_position_controller(void)
 
         const float stop_distance = stopping_distance();
 
-        if (poscontrol.get_state() == QPOS_AIRBRAKE && !(tiltrotor.enabled() && !tiltrotor.has_vtol_motor() && (tiltrotor.current_tilt >= tiltrotor.get_fully_forward_tilt()))) {
-            // don't ouput VTOL throttle on tiltrotors if there are no fixed VTOL motors and the tilt is still forward
+        if (!suppress_z_controller) {
+            // don't output VTOL throttle on tiltrotors if there are no fixed VTOL motors and the tilt is still forward
             hold_hover(0);
+        } else {
+            hold_stabilize(0.01);
```

Comment:
maybe a optional `force_airmode` bool that defaults false so we can output 0 throttle. functionally identical, but makes it clearer what the code is trying to do.

Reply:
Now were always outputting throttle it means that were always in `THROTTLE_UNLIMITED` so the check on line 2256 always puts you straight into `QPOS_POSITION1` skipping `QPOS_AIRBRAKE`

I think this whole if needs to be in a `tiltrotor.enabled()` check. Although I'm not sure we want to be always skipping airbrake on those either.

Reply:
we don't want to skip airbrake on tiltrotors. I think I have it right now, and will test on my Freeman2100 tomorrow